mrbgem.rake

```diff
@@ -49,13 +49,13 @@ MRuby::Gem::Specification.new('mruby-docopt') do |spec|
           run_command e, "sed -i -e 's/\\/usr\\/bin\\//\\/opt\\/osxcross\\/target\\/bin\\/x86_64-apple-darwin14-/' CMakeFiles/docopt_s.dir/link.txt"
         end
       end
-      run_command e, "make"
+      run_command e, "make docopt_s"
```

Comment:
Ah C. I guess I forgot to commit this change. On the plus side it also reduces the compile time. :)